design/README.md

```diff
@@ -0,0 +1,28 @@
+The budjen appliction needs to print a list of transactions in chronological order.
+
+Each class in the budjen application should one have one purpose (shown as =) and possess certain attributes (shown as -).
+
+Budjen class = run the application.
+- No fields.
+- One method called main, which will call the start method from the Application class.
+
+Application class = print a list of transactions in chronological order.
+- Four dependency fields called transactionData, orderer, transactionFormatter and logger.
+- One method called start that prints a list of transactions in chronological order.
+
+TransactionData class = provide the transaction data for the Application class.
+- One public method called getData.
+- One private method called makeTransaction to make a new transaction that sets the type, description, money and date.
+- One private method called credit to make a transaction where the transaction type is credit.
+One private method called debit to make a transaction where the transaction type is debit.
+
+Orderer class = order a given array by its dates in chronological order.
+- One public method called orderTheTransactions that will turn the given array into a list, sort the list by comparing each date to to next date, and then turn the list back into an array.
+
+TransactionFormatter class = define the format of the transaction.
+- One public method called format to provide the correct format.
+- One private method called formatPounds which will use the existing Currency class and provide a solution for the negative money.
```

Comment:
In other words, the `Currency` class is a dependency for this `TransactionFormatter` class.